src/xdocs/checks/javadoc/javadocpackage.xml

```diff
@@ -51,22 +51,101 @@
 
       <subsection name="Examples" id="Examples">
         <p id="Example1-config">
-          To configure the check:
+          To configure the check with no package-info.java file in directory:
         </p>
         <source>
 &lt;module name=&quot;Checker&quot;&gt;
   &lt;module name=&quot;JavadocPackage&quot;/&gt;
 &lt;/module&gt;
         </source>
+        <p id="Example1-structure">
+          <div class="wrapper">
+            <pre>
+src/xdocs-examples
+/resources/com/puppycrawl/tools/checkstyle/checks/javadoc/javadocpackage/nonlegacy/
+  Example1.java
```

Comment:
just `Example1.java`, full path is not required, html site readers dont know it.

Reply:
Done